CMakeLists.txt

```diff
@@ -702,6 +702,8 @@ include(version.cmake)
 
 find_path(ZMQ_INCLUDE_PATH zmq.hpp)
 find_library(ZMQ_LIB zmq)
+find_library(PGM_LIBRARY pgm)
+find_library(NORM_LIBRARY pgm)
```

Comment:
Are both of these lines supposed to say 'pgm'?

Reply:
:D